server/GRPC_TA2_TA3/relay.js

```diff
@@ -0,0 +1,58 @@
+// Javascript Imports
+const path = require("path");
```

Comment:
This file has mixed single quotes with double quotes. As VisFlow codebase uses single quotes consistently, maybe let's all use single quotes for javascript here.